scripts/AlternateImageVersion.ps1

```diff
@@ -13,7 +13,8 @@ if ( $Region -ne "" ) {
     Write-Host ("Warning: Region $Region is ignored")
 }
 
-Write-Host "Gateversion: $Gateversion"
+Write-Host "Gateversion1: $Gateversion Gatestack1: $Gatestack Gateversion2: $($Gateversion) Gatestack2: $($Gatestack)"
+Write-Host "Gateversion3: $env:Gateversion"
```

Comment:
If you were trying to access the AzureDevOps task variable Gate.Version's value then the syntax would be $ENV:GATE_VERSION.
And thats not required as its now working.